feature-libs/customer-ticketing/occ/adapters/occ-customer-ticketing.adapter.ts

```diff
@@ -80,7 +85,18 @@ export class OccCustomerTicketingAdapter implements CustomerTicketingAdapter {
     return this.http
       .get<TicketDetails>(this.getTicketEndpoint(customerId, ticketId))
       .pipe(
-        catchError((error) => throwError(normalizeHttpError(error))),
+        catchError((error) => {
+          if(error.error.errors[0].type === 'NotFoundError'
+              && error.error.errors[0].message.toLowerCase().startsWith('ticket'))
```

Comment:
How about doing something to avoid "error.error.error"
```suggestion
        catchError((errorResponse) => {
          const errorDetails = errorResponse.error.errors[0];
          if(errorDetails.type === 'NotFoundError'
              && errorDetails.message.toLowerCase().startsWith('ticket'))
```
In addition, why do you need to validate the message? Wouldn't be enough to maybe verify the return status:  if it is 404, then we know for sure a ticket with "ticketId" was not found.
But what happens if for XYZ reason: the customerId is not the owner of the ticket?

Reply:
same as above

Reply:
It is easier to add in the adapter as we are catching the error there, we might be able to pass the error to the component level as well.

But, as discussed with Ishrar, we can create request-handler as well, we are just not sure which if it would be an ideal solution in our scenario. to be on the safe side, we should take the opinion from Brian as he will be reviewing at the end.

Reply:
Thanks guys.. I have now added the check on http status code and also added the error details strings into a new module, which didn't previously exist.